api/odl.py

```diff
@@ -959,11 +956,12 @@ def run_once(self, progress):
         progress = TimestampData(achievements=message)
         return progress
 
+
 class MockODLAPI(ODLAPI):
     """Mock API for tests that overrides _get and _url_for and tracks requests."""
 
     @classmethod
-    def mock_collection(self, _db):
+    def mock_collection(cls, _db, protocol=ODLAPI.NAME):
```

Comment:
We don't need this right now -- it's for the ODL-via-OPDS-2 code that I didn't bring in -- but it doesn't hurt to have it.